app/views/healthcare/_allergies.html.slim

```diff
@@ -1,4 +1 @@
-= form_for form, url: form_path, method: :put do |f|
-  = f.radio_toggle_with_textarea :allergies
-
-  = render partial: 'buttons', locals: { f: f }
+= f.radio_toggle_with_textarea :allergies
```

Comment:
Should have been done in a different PR, but I removed an helper method from the controller and made the healthcare views more consistent with the ones we have in risks